src/styles/blocks/card.scss

```diff
@@ -0,0 +1,99 @@
+.card {
+  width: $card-width;
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  border: #f3f3f3 1px solid;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  &__description {
+    margin-top: 40px;
+  }
+  
+  &__image {
+    display: flex;
+    justify-content: center;
+
+    .image__imac {
```

Comment:
an element can't contain another element, according to BEM. Why don't set height and width to &__image?